### ISSUE TITLE
panic: package ./. already exists in cache 

### ISSUE THREAD

**Issue by grepsuzette** [gnolang/gno#638]
Occasionnaly I got `panic: package ./. already exists in cache`.
The first time I think I just resetted the blockchain (Edit: no lol, it happens during gnodev test). 
Are there any special tips to know about that?

**Comment by ltzmaxwell** [comment]
hi , can you give more details to reproduce this?

**Comment by r3v4s** [comment]
I don't remember exact message string but shouldn't gnodev return "package name wrong~~" if `_test` file is in different package??

As @ltzmaxwell mentioned, could you provide more detail info to reproduce

**Comment by grepsuzette** [comment]
I can not for now. In general there are many hard to understand errors returned by the VM when you write invalid code. Occasionnaly bugs.
I saw jae fixed a lot in his workshop PR. 

Valid code modulo the package name issues seems to work well however.
I just opened this bug because it was the 2nd time I faced it, but the fix was as indicated in the reply. 
I'll close this and reopen if it happens again. Thanks for your replies.
